app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -138,25 +124,40 @@ function getDocumentTypeLabel(documentType) {
 
 /**
  * @param {import('./acuant-capture-canvas').AcuantCaptureFailureError} error
+ * @param {string} code
  *
  * @return {string}
  */
-export function getNormalizedAcuantCaptureFailureMessage(error) {
+export function getNormalizedAcuantCaptureFailureMessage(error, code) {
   if (isAcuantCameraAccessFailure(error)) {
     return 'User or system denied camera access';
   }
 
-  if (!error) {
-    return 'Cropping failure';
+  const {
+    START_FAIL_CODE,
+    REPEAT_FAIL_CODE,
+    SEQUENCE_BREAK_CODE,
+  } = /** @type {AcuantGlobal} */ (window).AcuantJavascriptWebSdk;
+
+  switch (code) {
+    case START_FAIL_CODE:
+      return 'Camera failed to start (START_FAIL_CODE)';
+    case REPEAT_FAIL_CODE:
+      return 'Capture started after failure already occurred (REPEAT_FAIL_CODE)';
+    case SEQUENCE_BREAK_CODE:
+      return 'iOS 15 GPU Highwater failure (SEQUENCE_BREAK_CODE)';
+    default:
   }
 
   switch (error) {
+    case null:
+      return 'Cropping failure';
```

Comment:
This was removed from `AcuantCaptureFailureError` type. Double-check to see if it's still a possible value, else remove it here.

```suggestion
```

Reply:
> This was removed from `AcuantCaptureFailureError` type. Double-check to see if it's still a possible value, else remove it here.

`null` is not a possible value, but `undefined` is, used as an indication of cropping failure. Restored in f43b93f.